util/chplenv/chpl_libmode.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+import os
+import sys
+
+chplenv_dir = os.path.dirname(__file__)
+sys.path.insert(0, os.path.abspath(chplenv_dir))
+
+import overrides
+from utils import memoize
+
+
+
+@memoize
+def get():
+    libmode_val = overrides.get('CHPL_LIBMODE')
+    if not libmode_val:
+        libmode_val = 'none'
```

Comment:
Is it libmode `none`, or is there a better description? `notshared`, `private`? I feel like there's already a term, but I can't think of it off the top of my head.

Reply:
None is what we use as the default value today.  We could change that (and it's probably best to do that now before the PR gets merged).  Maybe `static` would be a better term?

Reply:
Huh, looks like we never look for anything other than `shared` for `CHPL_LIBMODE`, so really the default is "not shared".  I've been thinking of it as `none` in my head, probably because we use that in other places.

`notshared` seems a little weird for a default value.  `private` doesn't seem right to be (since we still access any symbol that is exported).  `position-dependent` seems a bit long. `unset` would maybe be confusing?  What about `off`?

Reply:
Yeah, I don't love any of these, especially my suggestions. Maybe just stick with `none`

Oh, actually what do we think of `default`?

Reply:
I'd be fine with `default` but it seems a little weird that we don't use it anywhere else (though nothing else is really a "mode" so maybe that's okay).  I'm tempted to run this by the larger group.

Reply:
For posterity, we had some offline discussion which resulted in the following changes:
- We are renaming the environment variable to better represent what it impacts.  It will now be `CHPL_LIB_PIC` instead of `CHPL_LIBMODE`.
- With that change, we have settled on `none` as the default and `pic` to represent the alternative behavior.